conu/apidefs/metadata.py

```diff
@@ -23,15 +32,32 @@ class ContainerMetadata(Metadata):
 
     def __init__(self, name=None, identifier=None, ports=None, labels=None, command=None,
                  creation_timestamp=None, env_variables=None,
-                 image=None, hostname=None, ipv4=None, ipv6=None, status=None):
+                 image=None, hostname=None, ipv4_addresses=None, ipv6_addresses=None, status=None):
+        """
+        :param name: str, name of container
+        :param identifier: int, id of container
+        :param ports: list, list of ports
```

Comment:
is this a list of exposed ports?

Reply:
Yes, the idea is to have list of exposed ports and then also dictionary which looks like {"ip_address": "port"}, Does it make sense to you? Or is enough to have IP addresses in list?

Reply:
+1 on having those as 2 separate fields

Reply:
e.g. `exposed_ports` and `port_mappings`